src/config.yaml

```diff
@@ -5,13 +5,19 @@ canopen_vcan:
     nodes: [{local: false, eds_location: 'eds_files/Arduino1.eds', node_purpose: {name: 'Front view object distance'
     , type: 0}}]
     max_speed_in_ms: 3
-    raw_can_data_logging: false
+    raw_can_data_logging: {
+        enabled: true,
+        buffer: 1024
```

Comment:
is dit 1024 can berichten?

Reply:
ja klopt ik zal wel een comment toevoegen later maak ik wel handleiding hiervoor

Reply:
ik zal wel een issue toevoegen hiervoor ^^